### ISSUE TITLE
Text reflow not working on some websites

### ISSUE THREAD

**Issue by notepaddo** [kiwibrowser/src.next#400]
Text reflow doesn’t work on the messages of the forum https://onlinephilosophyclub.com. It works fine in Yandex browser.

**Comment by KaKi87** [comment]
I cannot see any issue, can you post screenshots of what you get on Kiwi & Yandex browsers ?

**Comment by notepaddo** [comment]
I have enabled on both browsers the text wrap option. The follofing is what I am able get on Kiwi browser, on my Android smartphone. If I swipe horizontally the screen, it just doesn’t move, so, I can’t even try to see the trimmed text by swiping the screen.

![Screenshot_20210929-195600_Kiwi Browser](https://user-images.githubusercontent.com/14151656/135332434-2c26dce6-cbe5-4183-8b14-966137c7a4c9.jpg)

The following is what I am able get on Yandex browser. If I swipe the screen horizontally, the page moves, so that I can center it perfectly on the screen.

![Screenshot_20210929-195626_Browser](https://user-images.githubusercontent.com/14151656/135332626-9e5e9e53-c2db-42ce-b580-b7338781a93a.jpg)

**Comment by KaKi87** [comment]
Alright, I can reproduce, although horizontal scrolling is not fully blocked for me : it just glitches a bit but works if you're persistent.

I don't really know this feature, what is it for ? I see not much difference whether it's enabled or disabled, whether I'm zooming or not.

**Comment by notepaddo** [comment]
It makes a huge difference in a smartphone, because the screen is small and narrow and many websites have small fonts. So, if I zoom in without text wrap, I need to continuously swipe horizontally to read the the text that, because of the zooming, goes outside the screen on both sides. With text wrap, the lines are made shorter according to the screen and the words that would be at tbe end of the line outside the screen are instead moved to the next line, so I can see all the text in the screen and I need only to scroll vertically to follow the text.

**Comment by KaKi87** [comment]
What you're describing doesn't seem to happen on your second screenshot featuring Yandex Browser. Am I wrong ?

**Comment by notepaddo** [comment]
The picture shows that text wrapping works fine on Yandex: you can read at the center of the second screenshot “We all have free will, the freedom...”: Yandex moved the word “freedom” to the next line, because the action of zooming left no room in the first line. This way no words are missing. In the first screenshot, instead, which is from Kiwi, you can see that the words “the freedom to think on...” are outside the screen, I need to swipe horizontally to read them. On Yandex I don’t need to swipe horizontally: all the words of the phrase are there and I need just to scroll down to follow comfortably the text.

**Comment by KaKi87** [comment]
I see.

It seems to work on some lines but not all, like the second one from the top of your screenshot.

Also I would have thought it would preserve margins and borders.

**Comment by notepaddo** [comment]
Yes, even Yandex doesn’t manage properly the quoted text boxes, those ones with a grey/brown background color, I guess just because they are text boxes added in the middle of the flow of the main text. But, apart from this difficulty when additional text boxes are in the conversation, at least in Yandex I can read the message just scrolling it up and down, without needing to swipe horizontally.
